Add files via upload

eseigel opened the pull request:
index.ipynb - removed Colab chunk, formatted for consistency
index.Colab - made Colab notebook for users

eseigel committed:
Add files via upload

index.ipynb - removed Colab chunk, formatted for consistency
index.Colab - made Colab notebook for users

eseigel commented:
HI DONT APPROVE YET I don't have the Colab chunk in the Colab document

eseigel committed:
Add files via upload

Replaced index-Colab so it DOES have the Colab chunk for realsies this time

eseigel commented:
Ok I have added the Colab chunk to the new index-Colab file and now its only test fail is the normal exit error 1

HaleySchuhl commented:
Looking good so far. Thanks for tackling this, @eseigeldanf .

 I noticed in the error that it's saying `HTTP Error 404: Not Found` and the error is triggered at this line `img = pcv.io.open_url(url="
https://raw.githubusercontent.com/danforthcenter/plantcv-tutorial-color-correction/main/img/plant-image.jpg
")`. 

It looks like the actual test image for this tutorial is called `1_B73_sand_C_2023-04-14_10_19_07.jpg` so could you try updating the url path in that `open_url` line of code?

eseigel committed:
Add files via upload

Updated formatting for both index and index-Colab
Changed the image URL in Colab to be current
